src/resources/views/crud/fields/switch.blade.php

```diff
@@ -82,5 +82,17 @@ function bpFieldInitSwitch($element) {
     @endLoadOnce
 @endpush
 
+@if(\Str::startsWith($field['color'], '#'))
+@push('crud_fields_styles')
+    @loadOnce('bpFieldInitSwitchStyle')
+    <style>
+        .switch .switch-input:checked+.switch-slider {
+            background-color: {{ $field['color'] }};
+        }
+    </style>
+    @endLoadOnce
+@endpush
+@endif
+
```

Comment:
This isn't beauty, but it's needed for the hex background color.
Can't do it inline because it dependes on the `:checked`.

Reply:
Ouch! Doesn't this mean you can't have multiple switches in one form:
- one with color #a81913
- one with color #cca381

Am I reading this right @promatik ? If so... then no. I'd rather not have hex colors at all, than hex colors persisting across multiple fields. That is what we call "_a bug_" 😅

Reply:
I think we need to add it to the field as an attribute and then target the specific field, otherwise that CSS rule will target ALL checked switches.

Reply:
Ah true! I don't know how I didn't think about it 🤦‍♂️
So the only way would be, using Javascript ...

It can't be inline because of the `:checked+` css rule.
@pxpm do you see any other way?

Reply:
Solved with CSS variables 👌